questionnaires/views.py

```diff
@@ -0,0 +1,32 @@
+from django.shortcuts import get_object_or_404
+from django.http import Http404
+
+from dynamic_forms.models import FormModel
+from dynamic_forms.views import DynamicFormView
+
+from konfera.models import Ticket
+
+from .models import FormForTicket
+
+
+class ShowFormForTicket(DynamicFormView):
```

Comment:
@ricco386 I am having some difficulties with this class based view.

when I visit `/form/1/ticket/my_uuid/` the `dispatch` is invoked as supposed but when I submit the form, `form_valid` is not invoked, however the `form_valid` in `DynamicFormView` is. So basically I cannot override the method and I have no idea why. Any suggestions?